lib/fog/utilities.rb

```diff
@@ -1,7 +1,9 @@
 class Hash
   def symbolize_keys!
     keys.each do |key|
-      self[(key.to_sym rescue key)] = delete(key) if key.respond_to?(:to_sym) && !key.is_a?(Fixnum)
+      if key.respond_to?(:to_sym) && !key.is_a?(Integer)
```

Comment:
@plribeiro3000 possibly - but that could introduce nasty subtle bugs if users are expecting the keys to be symbols, and then this fix changes them to strings.